contrib/seeds/nodes_testnet4.txt

```diff
@@ -0,0 +1,9 @@
+18.189.156.102:48333 # AS16509
```

Comment:
nit: Typo in the commit message of 6c2a87fb1ccf26daa754e6d5a4c4ae687da562e3: "tesnet4"

Reply:
Fixed